server/app/models/fusor/deployment.rb

```diff
@@ -40,7 +40,7 @@ class Deployment < ActiveRecord::Base
     scoped_search :on => [:id, :name], :complete_value => true
 
     # used by ember-data for .find('model', {id: [1,2,3]})
-    scope :by_id, lambda { |n=nil| where(:id => n) if n.present? }
+    scope :by_id, proc { |n| where(:id => n) if n.present? }
 
```

Comment:
I would've left this as a separate change. As it is not purely a style change. After looking up proc and lambda, I'm ok with proc. But next time keep it separate.